packages/cisco_asa/data_stream/log/_dev/test/pipeline/test-repeat-deny.log-expected.json

```
@@ -0,0 +1,76 @@
+{
+  "expected": [
+    {
+        "cisco": {
+            "asa": {
+                "message_repeats": 2,
+                "source_interface": "inside"
+            }
+        },
+        "destination": {
+            "address": "10.1.2.42",
+            "ip": "10.1.2.42",
+            "port": 137
+        },
+        "ecs": {
+            "version": "8.11.0"
```

```suggestion
            "version": "8.17.0"
```

The ECS version has been updated on this integration since this was created, so you'll need to regenerate the pipeline test data for this test. Or you can just change the ECS version here, it's the only change in this case.

---

Updated